const numero1 = 20;
const numero3 = 30;

console.log(numero1);

try {//intentar
    console.log(numero2);
    
} catch (error) {//si no
    
}
console.log(numero3);